<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadYouTube</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Downloads/icons8-carpeta-de-películas-96.png</normaloff>../Downloads/icons8-carpeta-de-películas-96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #555;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>13</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 100pt &quot;Arial Black&quot;;
background-color: white;
border: 2px solid #ccc
</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_mp3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>Descargar MP3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_mp4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>Descargar MP4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
